src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/osDataCache.hpp

```diff
@@ -13,10 +13,16 @@
 #define _OS_DATA_CACHE_HPP
 
 #include "cacheLRU.hpp"
+#include "loggerHelper.h"
 #include "singleton.hpp"
+#include "socketDBWrapper.hpp"
+#include "vulnerabilityScannerDefs.hpp"
+#include "wazuhDBQueryBuilder.hpp"
 #include <shared_mutex>
 #include <string>
 
+auto constexpr OS_CACHE_WDB_SOCKET {"queue/db/wdb"};
```

Comment:
Maybe the constant WDB_LOCAL_SOCK from defs.h could be used directly without duplicating the definition.

Reply:
I'm afraid it wasn't possible.
The `defs.h` file contains an **USER** definition that conflicts with `rocksDB`

```
In file included from /home/vagrant/wazuh/src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/../scanOrchestrator/osDataCache.hpp:16,
                 from /home/vagrant/wazuh/src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/../scanOrchestrator/scanContext.hpp:19,
                 from /home/vagrant/wazuh/src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp:16,
                 from /home/vagrant/wazuh/src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.cpp:12:
/home/vagrant/wazuh/src/headers/defs.h:100:25: error: expected identifier before string constant
  100 | #define USER            "wazuh"
      |                         ^~~~~~~
/home/vagrant/wazuh/src/external/rocksdb/include/rocksdb/thread_status.h:47:5: note: in expansion of macro ‘USER’
   47 |     USER,               // User thread (Non-RocksDB BG thread)
      |     ^~~~
/home/vagrant/wazuh/src/headers/defs.h:100:25: error: expected ‘}’ before string constant
  100 | #define USER            "wazuh"
      |                         ^~~~~~~
/home/vagrant/wazuh/src/external/rocksdb/include/rocksdb/thread_status.h:47:5: note: in expansion of macro ‘USER’
   47 |     USER,               // User thread (Non-RocksDB BG thread)
      |     ^~~~
```